09_Going_Beyond/01_README.md

```diff
@@ -69,10 +69,35 @@ To implement a GUI our kernel requires:
 * To have a working graphic mode enabled. The `framebuffer` in our case (usually it is implemented by the bootloader)
 * To have functions to plot at least pixels, and probably basic shapes.
 * We should also have at least a font loaded and parsed, in order to be able to print some messages, and of course we need functions to print them. 
-* We need probably to have support for either keyboard or mouse (most probably we want to have a mouse driver)
+* We need probably to have support for either keyboard or mouse (most probably we want to have a mouse driver), and for the mouse is advisable to have a cursor pointer symbol loaded somewhere in memory. 
```

Comment:
One bit of general feedback: a lot of the language used here is very uncertain. I think we can be more sure of things like needing keyboard and mouse drivers.